wallet-sdk/src/main/kotlin/org/stellar/walletsdk/Anchor.kt

```diff
@@ -42,6 +45,42 @@ class Anchor(
       .await()
   }
 
+  /**
+   * Authenticate account with the anchor using SEP-10.
+   *
+   * @param accountAddress Stellar address of the account to authenticate
+   * @param clientDomain optional domain hosting stellar.toml file containing `SIGNING_KEY`
+   * @param memoId optional memo ID to distinguish the account
+   * @param toml Anchor's stellar.toml file containing `WEB_AUTH_ENDPOINT`
+   * @param walletSigner interface to define wallet client and domain (if using `clientDomain`)
+   * signing methods
+   *
+   * @return JWT auth token
+   */
+  suspend fun getAuthToken(
+    accountAddress: String,
+    clientDomain: String? = null,
+    memoId: String? = null,
+    toml: Map<String, Any>,
+    walletSigner: WalletSigner
+  ): String {
```

Comment:
Can you create a type alias (or better: inline class) for AuthToken?

Reply:
:+1: Let's do this when we update everything else.